pkgs/applications/science/logic/saw-tools/default.nix

```
@@ -53,7 +53,7 @@ stdenv.mkDerivation {
   meta = {
     description = "Tools for software verification and analysis";
     homepage    = "https://saw.galois.com";
-    license     = lib.licenses.unfreeRedistributable;
+    license     = licenses.bsd3;
```

```suggestion
    license     = lib.licenses.bsd3;
```

`licenses` is not in scope, thus causing an error.